### ISSUE TITLE
Loading bid for mixed banner/native adUnitId - Part 2

### ISSUE THREAD

**Issue by ArcaDone** [prebid/prebid-mobile-android#270]
Not being able to comment on the previous issue (Loading bid for mixed banner / native adUnitId # 269) I continue in this:

I have another question before considering the question closed: if we wanted to make two different requests by making two different calls (thus using two different AdUnitIDs, one for Native and one for Banner), there is a method in Prebid to understand what the price is . and then they choose the best? Is there a way to combine the different custom targeting?

Thanks in advance!

**Comment by YuriyVelichkoPI** [comment]
@ArcaDone, no, the SDK does not provide such a method yet. You should compare prices in the app. The price is available in the targeting keys. 

cc: @mittal-ank.

**Comment by YuriyVelichkoPI** [comment]
@ArcaDone are these issues still relevant? Can we help with your integration?
